test/integration/pavion.js

```diff
@@ -0,0 +1,90 @@
+/* global expect, chai, agent */
+
+const helpers = require('./helpers');
+
+/*
+ * The /pavions API endpoint
+ *
+ * This test suite implements full CRUD on the /projects HTTP API endpoint.
+ */
+describe('(/pavions) The pavion API endpoint', () => {
+  // project we will add during this test suite.
+  const uuid = '5b7dd0d6-9273-4955-a703-126fbd504b61';
+  const uuid2 = '7b7dd0d6-9273-4955-a703-126fbd504b61';
+
+  const pavion = {
+    uuid,
+    name : 'Pavion 1',
+    description : 'Frist pavion',
+    project_id : 1,
+    service_id : 1,
+  };
+
+  const pavion2 = {
+    uuid : uuid2,
+    name : 'Pavion 2',
+    description : 'Second pavion',
+    project_id : 1,
+  };
+
+  const pavionUpdate = {
+    uuid,
+    name : 'Pavion A',
+  };
+
+  it('POST /pavions add a new pavion', () => {
+    return agent.post('/pavions')
+      .send(pavion)
+      .then((res) => {
+        expect(res).to.have.status(201);
+      })
+      .catch(helpers.handler);
+  });
+
+  it('POST /pavions add pavion not linked to a service', () => {
+    return agent.post('/pavions')
+      .send(pavion2)
+      .then((res) => {
+        expect(res).to.have.status(201);
```

Comment:
You should also check that it returned an id/uuid.  You can use `helpers.api.created()` to help with this.